SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/folders.md

To retrieve information about the items contained in a folder, call
[`client.folders.listItems(folderId:usemarker:marker:offset:limit:sort:direction:fields:)`][get-folder-items]

> with the ID of the folder.  This method will return an iterator in the completion you can use to retrieve folder items.

Reviewer: ```suggestion
with the ID of the folder.  This method will return an iterator in the completion, which is used to retrieve folder items.
```

[copy-folder]: http://opensource.box.com/box-ios-sdk/Classes/FoldersModule.html#/s:6BoxSDK13FoldersModuleC10copyFolder8folderId011destinationF2ID4name6fields10completionySS_S2SSgSaySSGSgys6ResultOyAA0F0CAA0A5ErrorOGctF

> NOTE: Is this an iterator or not?

Reviewer: Is this a note for this code review?  Yes, the `listCollaborations` completion is:
`completion: @escaping Callback<PagingIterator<Collaboration>>`
Do you want to remove this comment?
